scheduler_error_mailer/models/ir_cron.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2012-2013 Akretion Sébastien BEAU,David Beal,Alexis de Lattre
+# Copyright 2016 Sodexis
+# Copyright 2018 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class IrCron(models.Model):
+    _inherit = "ir.cron"
+
+    email_template_id = fields.Many2one(
+        comodel_name="mail.template",
```

Comment:
```suggestion
        comodel_name="mail.template",
        domain=[("model_id.model", "=", "ir.cron")],
```
We can use the opportunity to add a domain.